### ISSUE TITLE
[SOLVED] MSI Afterburner with RivaTuner Statistic Server = QuickLook crashes

### ISSUE THREAD

**Issue by nfind** [QL-Win/QuickLook#555]
Hi, don't know if a Windows update is guilty for this, but since I upgraded my PC and done a fresh install of Windows 10 (1903) Quicklook stopped working. Neither the version of Github nor the version of the Microsoft Store work for me. A reinstall also does not work.

1. Launch Quicklook
2. Select any file to preview, the short "scrolling wheel appears" but nothing happens
3. Hovering over the Quicklook icon in the taskbar and it disappears
4. Can relaunch the app and even can right click on the icon in the taskbar to check for updates, whatever.

 - OS Version: Windows 10 (1903) 
 - QuickLook Version: 3.6.5 msi and Store version

Log from Windows Event-Viewer:

Name der fehlerhaften Anwendung: QuickLook.exe, Version: 3.6.5.0, Zeitstempel: 0x5ceab58f
Name des fehlerhaften Moduls: d3d9.dll, Version: 10.0.18362.329, Zeitstempel: 0x1ac9a115
Ausnahmecode: 0xc0000005
Fehleroffset: 0x00000000000ac44e
ID des fehlerhaften Prozesses: 0x199c
Startzeit der fehlerhaften Anwendung: 0x01d5792c5a31658a
Pfad der fehlerhaften Anwendung: C:\Users\~\AppData\Local\Programs\QuickLook\QuickLook.exe
Pfad des fehlerhaften Moduls: C:\Windows\SYSTEM32\d3d9.dll
Berichtskennung: 69a52439-6868-4f96-bc97-debea672f117
Vollständiger Name des fehlerhaften Pakets: 
Anwendungs-ID, die relativ zum fehlerhaften Paket ist: 
Quelle: Application Error

**Comment by xupefei** [comment]
Thanks. Is there any log at `%APPDATA%\pooi.moe\QuickLook\QuickLook.Exception.log` or `%LOCALAPPDATA%\Packages\21090PaddyXu.QuickLook_egxr34yet59cg\LocalCache\Roaming\pooi.moe\QuickLook\QuickLook.Exception.log`?

**Comment by nfind** [comment]
Hi! No there is no Log. I already checked before posting...

**Comment by xupefei** [comment]
I see on Google some other reports about this d3d9.dll error with Windows 10 1903.
Do you have MSI Afterburner installed?

**Comment by nfind** [comment]
Ah ok. Yes Afterburner is running on my system.

**Comment by xupefei** [comment]
According to https://github.com/digimezzo/Dopamine/issues/984#issuecomment-500374032, you may try to quit (or uninstall? I'm not sure) Afterburner and then run QuickLook.

**Comment by nfind** [comment]
Damn. Ok so it is the Afterburner which messed it up. Thanks a lot! But now I have to figure out how to correct this because with the Afterburner I control my fan setting of my graphics card. The DirectX installer directly from Microsoft just say "all is installed and nothing to do"...

**Comment by xupefei** [comment]
For fan controlling, You can try https://www.argusmonitor.com/  :)

**Comment by nfind** [comment]
Thanks. Or I just install the newest beta of the Afterburner and hope the problem is done, whatever. But good to know which causes the crash.

**Comment by nfind** [comment]
Ok. As I am not want to renounce the Afterburner, I did some advanced searching.. The d3d9.dll is not corrupt or something else but I found some older posts on the [Microsoft-Forum](https://answers.microsoft.com/en-us/windows/forum/games_windows_10/windows-10-directx-9-appsgames-all-fail-with/23ef2a8a-3cd4-4df3-af20-e87e10e334dc). Here someone find out that it is not the MSI Afterburner itself, its the RivaTuner Statistic Server which is normally shipped and auto-installed with the Afterburner. 
The solution is: Add the Quicklook.exe in the main window of RivaTuner and set the application detection level to none (see attached screenshot). After that Quicklook works great even when Afterburner + RTSS is running. This step has to be done with any other application which crashes on launch, when using Afterburner + RTSS. 
![RTSS_Quicklook](https://user-images.githubusercontent.com/24441569/66270437-f0b78780-e853-11e9-883d-f69bb73a2110.png)
